pkg/sdk/testint/external_functions_integration_test.go

```diff
@@ -194,6 +195,33 @@ func TestInt_ExternalFunctions(t *testing.T) {
 		assertExternalFunction(t, externalFunction.ID(), true)
 	})
 
+	t.Run("alter external function: set and unset tags", func(t *testing.T) {
```

Comment:
Maybe instead adding such a func to each object, we could extract one integration test with parametrized setup and run it for all objects instead? (I know that currently we had setting/unsetting tags in each object but with the rework, we can pick more centralized approach)

Reply:
I can do it in the next PR.

Reply:
Oky (keeping it open then)